features/collection/thresholding/impl/src/main/java/org/opennms/netmgt/threshd/AbstractThresholdEvaluatorState.java

```diff
@@ -151,5 +292,36 @@ protected String formatValue(double value) {
         return valueFormatter.format(value);
     }
 
+    @Override
+    public ThresholdingSession getThresholdingSession() {
+        return thresholdingSession;
+    }
+
+    private static byte[] serialize(Serializable value) {
```

Comment:
Just a note. We already have [fst](https://github.com/RuedigerMoeller/fast-serialization) in the class-path which may give us better performance here.